packages/utilities/src/focus.ts

```diff
@@ -196,11 +196,11 @@ export function isElementTabbable(element: HTMLElement): boolean {
 
   return (
     !!element && isFocusableAttribute !== 'false' &&
+    tabIndex >= 0 || !tabIndex &&
```

Comment:
We could exchange `tabIndex >= 0` with `!(tabIndex == -1)`.

The second `!tabIndex` is necessary to differentiate typically focusable elements from non-focusable elements when tabIndex is null.

Without `!tabIndex`, either: 
- `!(tabIndex == -1) &&` leaves all non A, Input, Button... elements as non-focusable  or  
- `!(tabIndex == -1) ||` leaves A, Input, Button... elements with tabindex="-1" still focusable (the original problem)